mw-webapp/src/styles/_variables.scss

```diff
@@ -13,3 +13,6 @@ $header-background: rgb(85, 122, 149);
 
 // Title
 $title-padding: 10px;
+
+// Font size
+$font-size-large: 48px;
```

Comment:
What about font size for title? Use mixin for title before merge

Reply:
1